guides/common/modules/ref_capsule-storage-requirements.adoc

```diff
@@ -5,7 +5,7 @@ The following table details storage requirements for specific directories.
 These values are based on expected use case scenarios and can vary according to individual environments.
 
 ifdef::katello,satellite[]
-The runtime size was measured with {RHEL} 6, 7, and 8 repositories synchronized.
+The runtime size was measured with {RHEL} 6, 7, 8 and 9 repositories synchronized.
```

Comment:
it was not :)

Reply:
It still gives the users a rough idea. At this point we should probably measure again with more recent versions, but it is an explanation of the third column.

Reply:
so here are a few raw numbers from the repos (I didn't sync them, so no idea if pulp will optimize anything, nor have an idea about the postgresql impact):
* RHEL9:
** baseos: 11GB
** appstream: 55GB
* RHEL8:
** baseos: 32GB
** appstream: 122GB
* RHEL7: 70GB

I think if we adjust the list to read "RHEL 7, 8, 9" the size will be in the right ballpark.

Reply:
@evgeni Addressed this :+1: